exads/examples/prebidJS-client-RTB-InStreamVideo.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang='en'>
+
+<head>
+    <meta charset='UTF-8'>
+    <title>RTB Instream Video Prebid.js</title>
+
+    <!-- Uncomment for development environments -->
+    <!-- <script src='js/prebid.js'></script> -->
+    <!-- Uncomment for production environments -->
+    <script async src='js/prebid.js' onload='onScriptLoaded()'></script>
+
+    <!-- Fluid Player CSS and JS -->
+    <script async src='https://cdn.fluidplayer.com/v3/current/fluidplayer.min.js'></script>
+    <script>
+        // Client Conf start
+        // Set as true for development environments only
+        const isEnabledDebug = false;
+        const exads_endpoint = 'https://your-ad-network.com/rtb.php';
+        const partner = 'rtb_2_4';
+        let zoneId = 5179865;
+        let fid = '12f6c5316c88687e275728a8f24f8787b6c31202';
+
+        // Start GDPR params
+        let isGDPRRequested = false;
+        // Valid iabtcf consent string 
+        let consentString = 'CO_V33gP0l1PgAoAAAENCZCgAAAAAAAAAAAAE0wAQE0gTTABATSAAA.YAAAAAAAAAA';
+        // End GDPR params
+
+        let siteId = '12345';
+        let country = 'IRL';
+        let keywords = 'lifestyle, humour';
+        let catIab = ['IAB25-3']; // if available - ex.: ['IAB1-5', 'IAB1-7']
+        let bidfloorcur = 'EUR';
+        let bidfloor = 0.00000011;
+        // Client Conf end
+        let adUnits = [];
+
+        let PREBID_TIMEOUT = 500000;
+        let userIp = 0;
+        let userIpLoaded = false;
+        let impression_id = hashCode(new Date().getTime().toString());
+        let user_id = localStorage.getItem('prebidJS.user_id');      
```

Comment:
most of const/variables are camelCase, so we should stick to it.

Reply:
done